### ISSUE TITLE
No audio in your intro video on Quick Start view

### ISSUE THREAD

**Issue by RandomFractals** [OfficeDev/TeamsFx#1378]
I tried to play your Capabilities intro video in the Quick Start welcome view, but there was no audio.

Is that just a gif?

**To Reproduce**

1. Install Teams Toolkit extension from VS Marketplace
2. Go to View -> Command Palette in vscode menu
3. Search for `Teams: Quick Start` command
4. Click on Watch Video in that welcome view.
5. Video plays but there is no audio. I had to `Watch on browser`. 

Also, I believe that label should read `Watch in a browser`

**Expected behavior**

Your Capabilities video plays with audio.

**Screenshots**

![image](https://user-images.githubusercontent.com/656833/122575413-58a50600-d016-11eb-9dd3-8895059833d8.png)

**VS Code Extension Information:**

Teams Toolkit (Preview) v2.1.0
VSCode Version: 1.57.1 (user setup)
Commit: 507ce72a4466fbb27b715c3722558bb15afa9f48
Date: 2021-06-17T13:28:07.755Z
Electron: 12.0.7
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Windows

**Additional context**

Most likely due to the fact that vscode can't really play a video with audio track. See the list of related issues in vscode repo: https://github.com/microsoft/vscode/issues/118275#issuecomment-791790131

However, until proper video codecs are added back to the Chromium version that vscode strips out, I would expect more transparent instructions on how to view your Capabilities video.

Thanks!

**Comment by zhenjiao-ms** [comment]
Thanks for reporting the issue. It is because VS code doesn't support audio when playing the video in its webview. When you move the mouse over the video, it will have a "Watch on browser" link, which can open the video in browser with audio.

**Comment by RandomFractals** [comment]
yes, as I pointed out in Additional context section, I am aware of that. 

I think your Watch on browser label should be changed and you should provider more transparent instructions in your intro video since it's devs first experience with your extension.

I do like the provided sidebar with links to docs, sample apps, and accounts. Well done!

**Comment by jjloki** [comment]
@RandomFractals  Thanks for your detailed feedback! Your feedback is very valuable to us. We will keep improving the user experiences based on your feedback in the following releases. Stay tuned!

**Comment by yanjiang222** [comment]
@RandomFractals Thanks for your detailed feedback! Your feedback is very valuable to us. We will keep improving the user experiences based on your feedback in the following releases. Stay tuned!

**Comment by RandomFractals** [comment]
Sounds good. Btw, I've built a few vscode extensions myself, mostly for data display. See my github profile.

I will be using your toolkit for creating some Teams apps and will provide more feedback as I develop them.

Great docs and toolkit extension so far.

Thanks!
